packages/php/blueprint/src/Cli/ExportCli.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Automattic\WooCommerce\Blueprint\Cli;
+
+use Automattic\WooCommerce\Blueprint\ExportSchema;
+use Automattic\WooCommerce\Blueprint\ZipExportedSchema;
+
+class ExportCli {
+	private string $save_to;
+	public function __construct($save_to) {
+		$this->save_to = $save_to;
+	}
+
+	public function run($args = array()) {
+		$export_as_zip = isset($args['format']) && $args['format'] === 'zip';
+
+		$exporter = new ExportSchema();
+
+		$schema = $exporter->export($args['steps'], $export_as_zip);
```

Comment:
Should we also check if the 'steps' key exists in the $args array before accessing it?

Reply:
Good catch! It's actually checked https://github.com/woocommerce/woocommerce/blob/add/blueprint-package-1/packages/php/blueprint/src/Cli.php#L51